FormalConjectures/Wikipedia/FibonacciPrimes.lean

```diff
@@ -16,13 +16,46 @@ limitations under the License.
 
 import FormalConjectures.Util.ProblemImports
 
+/-!
+# Fibonacci Primes
+
+*Reference:* [Wikipedia](https://en.wikipedia.org/wiki/Fibonacci_prime)
+-/
+
 /--
-This conjecture states that there are infinitely many Fibonacci primes
-(i.e., Fibonacci numbers that are prime). It is unsolved. It is
-stated on Wikipedia here: https://en.wikipedia.org/wiki/Fibonacci_prime.
+There are infinitely many Fibonacci primes, i.e., Fibonacci numbers that are prime
 It is also a barrier to defining a benchmark from this paper:
 https://arxiv.org/html/2505.13938v1 (see Figure 8).
+
+-/
+@[category research open, AMS 11]
+theorem fib_primes_infinite : {n : ℕ | (∃ m : ℕ, m.fib = n) ∧ n.Prime}.Infinite := by
+  sorry
+
+/--
+There are infinitely many indices $i$, such that the $i$-th Fibonacci is prime.
 -/
 @[category research open, AMS 11]
-theorem fib_primes_infinite : {n : ℕ | n.fib.Prime}.Infinite := by
+theorem fib_primes_infinite.variant : {n : ℕ | n.fib.Prime}.Infinite := by
```

Comment:
So how come you want this to be the variant?

Reply:
No, this doesn't matter so much to me either, but I guess what is your motivation for introducing the new variant?

Reply:
see discussion at #264